src/main/java/com/google/sps/data/Mentor.java

```diff
@@ -0,0 +1,39 @@
+package com.google.sps.data;
+
+import com.google.appengine.api.datastore.Entity;
+
+public class Mentor extends UserAccount {
+
+  public static final String ENTITY_TYPE = "Mentor";
+
+  private String dateOfBirth;
+  private String focus;
+  private String educationLevel;
```

Comment:
This again feels like something that should either be an enum or a richer field.

Reply:
This is changed to an enum now